pset-iterators.js

```diff
@@ -92,7 +103,11 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentialParties = presidents.map(president => {
```

Comment:
@anniecdavis27 great use of `.map()` to generate new array consisting only of party names